flow/ondemand/client/client.go

```diff
@@ -378,6 +378,21 @@ func (o *OnDemandProbeClient) Stop() {
 	o.checkForRegistration.Stop()
 }
 
+// Invoke capture upon start of analyzer on selected from config SubGraph
+func (o *OnDemandProbeClient) InvokeCaptureFromConfig() () {
```

Comment:
The extra presences for empty return value are redundant - so should be:

```
func (o *OnDemandProbeClient) InvokeCapturefromConfig() {
```

Reply:
u r right